README.md

```diff
@@ -254,3 +269,31 @@ redirect URI](https://learn.microsoft.com/en-us/entra/identity-platform/quicksta
 6. Save your changes
 
 </details>
+
+
+Development
+-----------
+
+__Features / TODO:__
+
+- [x] ability to set aws_region in env
+- [ ] ability to create and store favorites
+- [ ] support refresh tokens (kion currently doesn't use/support them)
+- [ ] possible to suport api key rotation?
+- [ ] support for other csps
+- [ ] pipeline to build for all platforms
+- [ ] pipeline to run unit tests
+- [ ] ability to copy staks to system clipboard
+- [ ] ability to federate into the console (open a browser tab) (waiting on dev fix)
+- [ ] ability to use upstream favorites defined in kion ui
+- [x] ability to auth with saml
+- [ ] ability to ssm into a host within an account
+- [ ] ability to set configs for multiple kion instances
+- [x] good documentation and in app help
+- [x] ability to use pre-defined favorites
+- [x] auto complete functions
+- [x] ability to generate short term tokens
+- [x] ability to use an api key (set via config file)
+- [x] ability to use username and password (passed via flags or propmted if no api key in config and no flags)
+- [x] makefile for building and installing into the path
+- [ ] example `.kion.yml` configuration file, or have the application create it if it doesn't exist and prompt the user for the required values to ease initial setup!
```

Comment:
Also from an old version.  It may be best to clone a new version of the repo from Github only, apply your changes to that version, and upload a new PR from there.